nav2_system_tests/src/updown/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+ament_add_gtest_executable(test_updown
+  test_updown.cpp
+)
+
+ament_target_dependencies(test_updown
+  ${dependencies}
+)
+
+ament_add_test(test_updown
+  GENERATE_RESULT_FOR_RETURN_CODE_ZERO
+  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/start_nav2"
```

Comment:
Can this ever fail? I don't see any way for the launch script to exit with some sort of status code like our other tests.

Reply:
I've removed the ament_add_test since this is just a simple stress test that doesn't need to be run with CI.